kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java

```diff
@@ -26,31 +26,24 @@
    */
   private final RestMetricsContext metricsContext;
 
-  public static final String KAFKA_REST_RESOURCE_TYPE = "kafka.rest";
   public static final String RESOURCE_LABEL_PREFIX = "resource.";
-  public static final String RESOURCE_LABEL_COMMIT_ID = RESOURCE_LABEL_PREFIX + "commit.id";
   public static final String RESOURCE_LABEL_TYPE = RESOURCE_LABEL_PREFIX + "type";
   public static final String RESOURCE_LABEL_VERSION = RESOURCE_LABEL_PREFIX + "version";
+  public static final String RESOURCE_LABEL_COMMIT_ID = RESOURCE_LABEL_PREFIX + "commit.id";
+  public static final String RESOURCE_LABEL_CLUSTER_ID = RESOURCE_LABEL_PREFIX + "kafka.cluster.id";
+
+  public static final String KAFKA_REST_RESOURCE_TYPE = "kafka_rest";
+  public static final String KAFKA_REST_RESOURCE_CLUSTER_ID_DEFAULT = "cluster_id_unavailable";
 
   public KafkaRestMetricsContext(String namespace, Map<String, Object> config) {
     metricsContext = new RestMetricsContext(namespace, config);
 
-    setResourceLabel(RESOURCE_LABEL_COMMIT_ID, AppInfoParser.getCommitId());
     setResourceLabel(RESOURCE_LABEL_TYPE, KAFKA_REST_RESOURCE_TYPE);
+    setResourceLabel(RESOURCE_LABEL_CLUSTER_ID,
+            (String) config.getOrDefault("bootstrap.cluster.id",
```

Comment:
I'm curious how this config gets defined? I couldn't find it in the codebase.

Reply:
It's a follow-up PR coming to  ce-kafka-rest. I'll mark you as reviewer there as well. We  had to make a few changes with regard to how the  servlet was discovered/loaded registered with the embedded http server first but those have now been merged. I'll DM you the companion PR tomorrow morning.

Reply:
https://github.com/confluentinc/ce-kafka-rest/pull/37